jsonnet/dashboards/apps/prometheus-dashboard.libsonnet

```diff
@@ -0,0 +1,408 @@
+        );
+
+      local scrapeRej =
+        graphPanel.new(
+          title='Rejected Scrapes',
+          datasource='$datasource',
+          labelY1='Scrapes',
+          min=0,
+        )
+        .addTargets(
+          [
+            prometheus.target('sum(prometheus_target_scrapes_exceeded_sample_limit_total{job=~"$job",instance=~"$instance"})', legendFormat='exceeded sample limit'),
+            prometheus.target('sum(prometheus_target_scrapes_sample_duplicate_timestamp_total{job=~"$job",instance=~"$instance"})', legendFormat='exceeded sample limit'),
+            prometheus.target('sum(prometheus_target_scrapes_sample_out_of_bounds_total{job=~"$job",instance=~"$instance"})', legendFormat='out of bounds'),
+            prometheus.target('sum(prometheus_target_scrapes_sample_out_of_order_total{job=~"$job",instance=~"$instance"})  ', legendFormat='out of order'),
+          ]
+        );
+
+      local engQueDur =
+        graphPanel.new(
+          title='Prometheus Engine Query Duration Seconds',
+          datasource='$datasource',
+          min=0,
+          labelY1='Seconds',
+        )
+        .addTarget(prometheus.target('sum(prometheus_engine_query_duration_seconds_sum{job=~"$job",instance=~"$instance"}) by (slice)', legendFormat='{{slice}}'));
+
+      local notSent =
+        graphPanel.new(
+          title='Notifications Sent',
+          datasource='$datasource',
+          min=0,
+          labelY1='Notifications',
+        )
+        .addTarget(prometheus.target('rate(prometheus_notifications_sent_total[5m])', legendFormat='{{instance}}'));
+
+      local minConf =
+        graphPanel.new(
+          title='Minutes Since Successful Config Reload',
+          datasource='$datasource',
+          min=0,
+          labelY1='Minutes',
+        )
+        .addTarget(prometheus.target('(time() - prometheus_config_last_reload_success_timestamp_seconds{job=~"$job",instance=~"$instance"}) / 60', legendFormat='{{instance}}'));
+
+      local succConf =
+        graphPanel.new(
+          title='Successful Config Reload',
+          datasource='$datasource',
+          min=0,
+          labelY1='Success',
+        )
+        .addTarget(prometheus.target('prometheus_config_last_reload_successful{job=~"$job",instance=~"$instance"}', legendFormat='{{instance}}'));
+
+      local gcRate =
+        graphPanel.new(
+          title='GC Rate / 2m',
+          datasource='$datasource',
+        )
+        .addTarget(prometheus.target('sum(rate(go_gc_duration_seconds_sum{instance=~"$instance",job=~"$job"}[2m])) by (instance)', legendFormat='{{instance}}'));
+
+      local appSampleRate =
+        graphPanel.new(
+          title='Appended sample rate',
+          datasource='$datasource',
+          labelY1='New samples appended',
+          min=0,
+        )
+        .addTarget(prometheus.target('rate(prometheus_tsdb_head_samples_appended_total[$interval])', legendFormat='samples appended to db By sec'));
+
+      local retNeeded =
```

Comment:
This panel is [not working](https://monitoring.k8s-h-dev.dnation.cloud/d/prometheus/prometheus?orgId=1&refresh=10s) for me: 
![image](https://user-images.githubusercontent.com/72123207/164221834-69391980-c546-4574-b766-e456184b9f4e.png)
`bad_data: 1:2: parse error: unexpected <op:*>`